src/cache/cache_las.c

```diff
@@ -18,6 +18,25 @@
 	(WT_SESSION_IGNORE_CACHE_SIZE | WT_SESSION_READ_WONT_NEED |	\
 	WT_SESSION_NO_RECONCILE)
 
+/*
+ * __las_timing_stress --
+ *	Optionally add delay to simulate the race conditions in lookaside
+ * sweep for debug purposes. The purpose is to uncover the race conditions in
+ * lookaside sweep.
```

Comment:
These two sentences say nearly the exact same thing. I suggest removing the 2nd sentence.